frontend/src/pages/CreateUser.tsx

```diff
@@ -0,0 +1,211 @@
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { connect } from "react-redux";
+import { useHistory } from "react-router-dom";
+import styled from "styled-components";
+import Layout from "../components/Layout";
+import { TextInput } from "../components/TextInput";
+import { INK, LOGIN_BACKGROUND } from "../constants/colors";
+import { authenticationRequest } from "../data/actions";
+import { AccountType, CreateUserParams } from "../models/types";
+
+const LoginHoldingDiv = styled.div`
+  display: flex;
+  align-items: stretch;
+  flex-direction: column;
+  @media (max-width: 600px) {
+    flex: 2;
+    justify-content: flex-start;
+  }
+
+  @media (min-width: 601px) {
+    flex: 1;
+    justify-content: center;
+  }
+`;
+
+const CreateUserSwitchingDiv = styled.div`
+  display: flex;
+  align-items: stretch;
+  min-height: 100%;
+  @media (max-width: 600px) {
+    flex-direction: column;
+  }
+
+  @media (min-width: 601px) {
+    flex-direction: row;
+  }
+`;
+
+const ActuallyCreateUserDiv = styled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const LoginInfoDiv = styled.div`
+  flex: 1;
+  background: ${LOGIN_BACKGROUND};
+`;
+
+const HorizontalDiv = styled.div`
+  display: flex;
+  flex-direction: row;
+`;
+
+const InputDiv = styled.div`
+  justify-content: center;
+  align-content: center;
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+`;
+const InputText = styled.p`
+  margin: 2px;
+  text-align: center;
+  flex: 1;
+`;
+
+const RadioButton = styled.input`
+  flex: 1;
+  margin: auto;
+`;
+
+const SignUpHeader = styled.h1``;
+
+const EvenSpacedDiv = styled.div`
+  @media (max-width: 600px) {
+    flex: 0;
+  }
+
+  @media (min-width: 601px) {
+    flex: 1;
+  }
+`;
+
+const CreateUserButton = styled.button`
+  background-color: ${INK};
+  border-radius: 12px;
+  color: #fff;
+  border-width: 0px;
+  flex: 1;
+  margin: 5px 0px 15px 0px;
+  padding: 15px;
+`;
+
+// An example of using a connector
+const connector = connect((state) => state, {
+  createUser: authenticationRequest.createUser,
+});
+
+interface CreateUserProps {
+  signedIn: boolean;
+  createUser: ({ email, password, name, accountType, age } : CreateUserParams) => void 
+}
+
+const CreateUser : FunctionComponent<CreateUserProps> = ({ signedIn, createUser }) : ReactElement => {
+  let [email, setEmail] = useState("");
+  let [password, setPassword] = useState("");
+  let [confirmPassword, setConfirmPassword] = useState("");
+  let [name, setName] = useState("");
+  let [accountType, setAccountType] = useState("");
+  let [age, setAge] = useState("");
+  let createUserWithCheck = () => {
+    if (confirmPassword !== password) {
+      alert("You need to confirm the password with the same password");
```

Comment:
Actually let's use antd's package Notification for alert :))

Reply:
Oh didn't know it has it, might do that in the style pr as well, since that seems like a styling type of thing. Do you agree?

Reply:
okie that makes sense :))